internal/action/repl.go

```diff
@@ -13,6 +13,31 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+type mindFlagsCompleter struct {
+	base readline.AutoCompleter
+}
+
+func (f *mindFlagsCompleter) Do(line []rune, pos int) ([][]rune, int) {
+	// Get the portion of the line up to the cursor and split to tokens
+	origInput := string(line[:pos])
+	tokens := strings.Fields(origInput)
+	var filtered []string
```

Comment:
Make it `filtered := make([]string, 0, len(tokens))`, please. This avoids resizing the underlying array later. It's probably not performance relevant, but for consistency. Thanks.